proposals/specification/interop.shex

```diff
@@ -56,6 +56,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
   interop:registeredAt xsd:dateTime ;
   interop:updatedAt xsd:dateTime ;
   interop:registeredShapeTree IRI ;
+  interop:registeredType IRI* ;
```

Comment:
Probably worth a discussion about whether this registeredType should be required or not.